### PR TITLE
Add v13.0.0 for Azure

### PR DESCRIPTION
**Pull request by MarcelMue**

### CODE REVIEW

File: azure/v13.0.0/README.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+
+
+### external-dns [1.5.0](https://github.com/giantswarm/external-dns-app/releases/tag/v1.5.0)
+
+#### Changed
+- Upgrade upstream external-dns from v0.7.3 to [v0.7.4](https://github.com/kubernetes-sigs/external-dns/releases/tag/v0.7.4).
+
+
+
+### cluster-autoscaler [1.18.2](https://github.com/giantswarm/cluster-autoscaler-app/releases/tag/v1.18.2)
```
Comment (fiunchinho) [inline azure/v13.0.0/README.md:180]: Should I remove cluster-autoscaler from the readme?

Comment (MartaD) [reply azure/v13.0.0/README.md:180]: yes

**Comment by T-Kukawka** [comment]
looks good to me 👍

File: azure/v13.0.0/README.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
+# :zap: Giant Swarm Release v13.0.0 for Azure :zap:
+
+This is the first release to support Kubernetes 1.18 on Azure.
+
+This release also includes support for Kubernetes node pools.
+A node pool is a sub set of the kubernetes nodes, and node pools differ from each other.
+This is suitable for having different node pools with different configurations (like a different instance size) for different purposes.
+After cluster creation with 1 node pools, later on node pools can be freely added and removed from the cluster.
```
Comment (MartaD) [inline azure/v13.0.0/README.md:8]:
```suggestion
After cluster creation with 1 node pools, additional node pools can be freely added and removed from the cluster.
```

File: azure/v13.0.0/README.md [hunk 3]
```diff
@@ -0,0 +1,178 @@
+# :zap: Giant Swarm Release v13.0.0 for Azure :zap:
+
+This is the first release to support Kubernetes 1.18 on Azure.
+
+This release also includes support for Kubernetes node pools.
+A node pool is a sub set of the kubernetes nodes, and node pools differ from each other.
+This is suitable for having different node pools with different configurations (like a different instance size) for different purposes.
```
Comment (MartaD) [inline azure/v13.0.0/README.md:7]:
```suggestion
A node pool is a subset of the Kubernetes nodes. They enable having pools of nodes with different configurations (like a different instance size) within one cluster.
```

File: azure/v13.0.0/README.md [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+Using [our kubectl plugin](https://github.com/giantswarm/kubectl-gs/) you can easily create the Custom Resources required to create a cluster.
+
+## Change details
+
+### azure-operator [5.0.0](https://github.com/giantswarm/aws-operator/releases/tag/v5.0.0)
+
+Bug fixes:
+
+- Fixed firewall rules to allow prometheus to scrape node-level exporters from all node pools.
+- Encryption secret is now taken from the CR namespace rather than the organization namespace.
```
Comment (MartaD) [inline azure/v13.0.0/README.md:21]: what customers get from this change?

File: azure/v13.0.0/README.md [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+- Try to send only one request to VMSS Azure API from `nodepool` handler.
```
Comment (MartaD) [inline azure/v13.0.0/README.md:22]:
```suggestion
- Try to send only one request to VMSS Azure API from `nodepool` handler which reduces the risk of reaching API rate limit and getting 429 errors.
```

File: azure/v13.0.0/README.md (hunk 4 above)
Comment (fiunchinho) [reply azure/v13.0.0/README.md:21]: Probably it's too low level. I'll remove it.

**Review by MartaD — APPROVED** [review]

**Comment by MarcelMue** [comment]
I assume this isn't needed anymore.

**Pull request closed without merging**